### PR TITLE
For the head not the food

### PR DESCRIPTION
**Pull request by Aurorablade**
:cl: Fethas
bugfix: Though shalt not eat robotic organs..including cybernetic implants..
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
*thou

**Comment by Iamgoofball** [comment]
also you forgot the CL ending marker

/:cl:

**Comment by Crazylemon64** [comment]
Mmm, high in iron

File: code/modules/surgery/organs/organ_internal.dm [hunk 1]
```diff
@@ -76,6 +76,8 @@
 	return ..()
 
 /obj/item/organ/internal/proc/prepare_eat()
+	if(robotic >= 2)
```
Comment (TheDZD) [inline code/modules/surgery/organs/organ_internal.dm]: Might just want to have this be `if(robotic)` so that you can't eat assisted organs either.

Comment (Aurorablade) [inline code/modules/surgery/organs/organ_internal.dm]: assisted is robotic 1..so they might just peel the assited parts off like an organge

**Pull request merged**